src/components/ErrorAlert/ErrorAlert.component.tsx

```diff
@@ -0,0 +1,58 @@
+import { useCallback, useEffect, useRef } from 'react';
+import { Alert, AlertTitle } from '@mui/material';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { errorMessageSelector } from '../../store/app/app.selectors';
+import { appActions } from '../../store/app/app.slice';
+
+export const ErrorAlert = () => {
+  const dispatch = useDispatch();
+  const message = useSelector(errorMessageSelector);
+  const handlerRef = useRef<NodeJS.Timeout>();
+
+  const handleClose = useCallback(() => {
+    dispatch(appActions.setErrorMessage(undefined));
+  }, [dispatch]);
+
+  useEffect(() => {
+    if (handlerRef.current) {
+      clearTimeout(handlerRef.current);
+      handlerRef.current = undefined;
+    }
+
+    if (message) {
+      handlerRef.current = setTimeout(handleClose, 5000);
```

Comment:
We could also use the `Snackbar` component here:
https://mui.com/material-ui/api/snackbar/

```typescript
        <Snackbar
          open={!!message}
          onClose={handleClose}
          autoHideDuration={NOTIFICATION_DURATION}
          ClickAwayListenerProps={{ onClickAway: () => {} }}
        >
          <Alert
            severity="error"
            variant="filled"
            onClose={handleClose}
          >
            {message}
          </Alert>

```